cmd/titus/notion.go

```
@@ -0,0 +1,197 @@
+				}
+			}
+		}
+
+		return nil
+	})
+
+	if err != nil {
+		return fmt.Errorf("scanning Notion: %w", err)
+	}
+
+	fmt.Fprintf(cmd.ErrOrStderr(), "Notion scan complete: %d matches, %d findings\n", matchCount, findingCount)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)